packages/ember-data/lib/serializers/json-api-serializer.js

```diff
@@ -58,12 +58,12 @@ var dasherize = Ember.String.dasherize;
               "data": [
                 {
                   "id": "3",
-                  "type": "players"
+                  "type": "player"
```

Comment:
This is confusing, at least for me. Shouldn't this be mention somewhere, somehow?